lighthouse-core/scripts/esm-utils.js

```diff
@@ -6,13 +6,31 @@
 'use strict';
 
 import module from 'module';
+import url from 'url';
+import path from 'path';
 
 const require = module.createRequire(import.meta.url);
 
 /**
  * Commonjs equivalent of `require.resolve`.
  * @param {string} packageName
  */
-export function resolveModulePath(packageName) {
+function resolveModulePath(packageName) {
   return require.resolve(packageName);
 }
+
+/**
+ * Commonjs equivalent of `require.resolve`.
```

Comment:
nit: this looks like it was copied from `resolveModulePath`, does it make sense here as well?